contracts/0.4.24/interfaces/ILido.sol

```diff
@@ -4,111 +4,114 @@
 
 pragma solidity 0.4.24;
 
-
 /**
-  * @title Liquid staking pool
-  *
-  * For the high-level description of the pool operation please refer to the paper.
-  * Pool manages withdrawal keys and fees. It receives ether submitted by users on the ETH 1 side
-  * and stakes it via the deposit_contract.sol contract. It doesn't hold ether on it's balance,
-  * only a small portion (buffer) of it.
-  * It also mints new tokens for rewards generated at the ETH 2.0 side.
-  *
-  * At the moment withdrawals are not possible in the beacon chain and there's no workaround.
-  * Pool will be upgraded to an actual implementation when withdrawals are enabled
-  * (Phase 1.5 or 2 of Eth2 launch, likely late 2022 or 2023).
-  */
+ * @title Liquid staking pool
+ *
+ * For the high-level description of the pool operation please refer to the paper.
+ * Pool manages withdrawal keys and fees. It receives ether submitted by users on the ETH 1 side
+ * and stakes it via the deposit_contract.sol contract. It doesn't hold ether on it's balance,
+ * only a small portion (buffer) of it.
+ * It also mints new tokens for rewards generated at the ETH 2.0 side.
```

Comment:
This description is outdated (see the notes for `Lido.sol`)

Reply:
fixed